.gitignore

```diff
@@ -31,3 +31,5 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+.env.*
```

Comment:
```suggestion
.env.*

```

Reply:
More context, here's the definition of line: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206